### PR TITLE
at/cb-122 Implement inubekit on data components

### PR DESCRIPTION
**Pull request by Angelbt12**
Implement inubekit on data components
image

### CODE REVIEW

**Review by Brandonz07 — APPROVED** [review]

File: src/components/data/Fieldset/styles.ts [hunk 1]
```diff
@@ -15,19 +15,19 @@ export const StyledContainerFieldset = styled.div<IStyledContainerFieldset>`
...
+    theme?.palette?.neutral?.N200 || inube.palette.neutral.N200};
   box-shadow: 0px 2px 6px
     ${({ theme }) =>
-      theme?.color?.stroke?.divider?.regular ||
-      inube.color.stroke.divider.regular};
+      theme?.palette?.neutral?.N40 ||
+      inube.palette.neutral.N40};
   padding-top: ${({ theme, $hasTable }) =>
-    !$hasTable && (theme?.spacing?.s200 || inube.spacing.s200)};
+    !$hasTable && (theme? "16px" : "16px" )};
```
Comment (carin91) [inline src/components/data/Fieldset/styles.ts:24]: Delete the validation from theme and pass directly the 16px

File: src/components/data/Fieldset/styles.ts [hunk 2]
```diff
@@ -15,19 +15,19 @@ export const StyledContainerFieldset = styled.div<IStyledContainerFieldset>`
...
   padding-bottom: ${({ theme, $hasTable }) =>
-    !$hasTable && (theme?.spacing?.s200 || inube.spacing.s200)};
+    !$hasTable && (theme? "16px" : "16px" )};
```
Comment (carin91) [inline src/components/data/Fieldset/styles.ts:26]: Delete the validation from theme and pass directly the 16px

File: src/components/data/Fieldset/styles.ts [hunk 3]
```diff
@@ -15,19 +15,19 @@ export const StyledContainerFieldset = styled.div<IStyledContainerFieldset>`
...
   padding-right: ${({ theme, $hasTable }) =>
-    !$hasTable && (theme?.spacing?.s100 || inube.spacing.s100)};
+    !$hasTable && (theme? "8px" : "8px" )};
```
Comment (carin91) [inline src/components/data/Fieldset/styles.ts:28]: Delete the validation from theme and pass directly the 8px

File: src/components/data/Fieldset/styles.ts [hunk 4]
```diff
@@ -15,19 +15,19 @@ export const StyledContainerFieldset = styled.div<IStyledContainerFieldset>`
...
   padding-left: ${({ theme, $hasTable }) =>
-    !$hasTable && (theme?.spacing?.s100 || inube.spacing.s100)};
+    !$hasTable && (theme? "8px" : "8px" )};
```
Comment (carin91) [inline src/components/data/Fieldset/styles.ts:30]: Delete the validation from theme and pass directly the 8px

**Review by carin91 — APPROVED** [review]

File: src/components/data/Fieldset/styles.ts (hunk 2 above)
Comment (Angelbt12) [reply src/components/data/Fieldset/styles.ts:26]: ok

File: src/components/data/Fieldset/styles.ts (hunk 1 above)
Comment (Angelbt12) [reply src/components/data/Fieldset/styles.ts:24]: ok

File: src/components/data/Fieldset/styles.ts (hunk 2 above)
Comment (Angelbt12) [reply src/components/data/Fieldset/styles.ts:26]: ok

File: src/components/data/Fieldset/styles.ts (hunk 3 above)
Comment (Angelbt12) [reply src/components/data/Fieldset/styles.ts:28]: ok

File: src/components/data/Fieldset/styles.ts (hunk 4 above)
Comment (Angelbt12) [reply src/components/data/Fieldset/styles.ts:30]: ok

**Pull request merged**